### PR TITLE
Added pathplanner paths as well as autos to run them

### PR DESCRIPTION
**Pull request by stardust4089**
@GreenMachineStudios created the paths and put them into template autos. The paths have not been tested on a practice field yet. 

Includes:
- Path to move from starting location to the charging station (Preload+park)
- Path to score preload, grab one, score it. 
- Identical path to above, but for opposite starting position

These are then inverted in the code for the red side autos.

If you wish to review the paths, opening the branch in pathplanner on your computer will show them. 

I believe that while not tested on a field, these are ready to go on the robot because it will allow us to easier move forward with making autos for this weekend's practice field session. They can be tuned on the field and those tunings will be merged in after.

Also includes LevelRobot and Photonvision.

### CODE REVIEW

File: src/main/java/frc/robot/commands/SwerveControllerFollower.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        // SmartDashboard.putNumber("Target Heading",
+        // targetPose.getRotation().getDegrees());
+        // SmartDashboard.putNumber("Target X", targetPose.getX());
+        // SmartDashboard.putNumber("Target Y", targetPose.getY());
+    }
+
+    // Called once the command ends or is interrupted.
+    @Override
+    public void end(boolean interrupted) {
+        timer.stop();
```
Comment (mikemag) [inline src/main/java/frc/robot/commands/SwerveControllerFollower.java:93]: If this command is interrupted, won't the drivetrain still be moving (possibly rapidly) from the last call to `drivetrain.drive()`? Feels like you wanna tell the drivetrain to stop if the command is interrupted.

**Review by mikemag — CHANGES_REQUESTED** [review]
There are a lot of changed files outside the scope of this PR. Would a merge w/ main help that? Or is it intentional? I think the auto commands are fine starts, and have one comment on the follower commend. I didn't really look at the rest in the hope this PR will get smaller. Let me know!

**Review by mikemag — COMMENTED** [review]
Also, I think you should add the `generatedJSON` directory to your .gitignore. It looks like these are auto-generated Pathweaver JSON files (and they're huge). But you're only using the PathPlanner files.

**Review by mikemag — CHANGES_REQUESTED** [review]
There's still two files left in `generatedJSON`.

I don't see the change to `end` for `SwerveTrajectoryFollowCommand`

I think the other changes are fine. Consider updating the PR description to note this includes PhotonVision and leveling.

File: src/main/java/frc/robot/commands/SwerveControllerFollower.java (hunk 1 above)
Comment (stardust4089) [reply src/main/java/frc/robot/commands/SwerveControllerFollower.java]: fixed!

**Comment by stardust4089** [comment]
> There's still two files left in `generatedJSON`.
> 
> I don't see the change to `end` for `SwerveTrajectoryFollowCommand`
> 
> I think the other changes are fine. Consider updating the PR description to note this includes PhotonVision and leveling.

Fixed!

**Review by mikemag — APPROVED** [review]

**Review by SamarthRao91 — APPROVED** [review]
Went through it breifly and it looked solid. 👍

**Review by mikemag — APPROVED** [review]
New changes look fine.

Recommend not adding new little changes like this to an already approved PR. These could have been their own thing easily.

**Pull request merged**